File: DailyFlash Solutions/DailyFlash Solutions/07-DailyFlash_Solutions/21_Jan_Solutions_One/Cpp/pattern.cpp
/*Program 5 : Write a Program to print following Pattern.
Output :
0
0 0 
0 0 0
0 0 0 0*/


#include<iostream>

#define LOWERLIMIT 0
#define UPPERLIMIT 4

int main(){
	
	for(int outeritr = LOWERLIMIT ; outeritr < UPPERLIMIT ; outeritr++){
		
		for(int inneritr =  LOWERLIMIT ; inneritr <=outeritr ; inneritr++){
			
			printf("0 ");
		}	
		printf("\n");
	}	
}	

File: DailyFlash Solutions/DailyFlash Solutions/07-DailyFlash_Solutions/21_Jan_Solutions_One/Java/Pattern.java
/*Program 5 : Write a Program to print following Pattern.
Output :
0
0 0 
0 0 0
0 0 0 0*/


class Pattern{

	public static void main(String args[]){

		for(int outeritr = 0 ; outeritr < 4 ; outeritr++){
		
			for(int inneritr =  0 ; inneritr <=outeritr ; inneritr++){
			
				System.out.printf("0 ");
			}	
			System.out.printf("\n");
		}

	}	
}	
